flink-table/flink-table-storage/src/test/java/org/apache/flink/table/storage/filestore/manifest/ManifestTest.java

```diff
@@ -55,7 +55,7 @@ public void testWriteAndReadManifestFile() throws IOException {
                         TestKeyValueGenerator.PARTITION_TYPE,
                         TestKeyValueGenerator.ROW_TYPE,
                         TestKeyValueGenerator.KEY_TYPE,
-                        new TestOrcFileFormat());
+                        new TestJsonFileFormat());
```

Comment:
Can you use avro instead? I think we should use avro for manifest file finally

Reply:
Then we'll have to rebase FLIP-188 onto the latest community master. Would you please update that branch?